practice-three/src/style/page/introduce.css

```diff
@@ -0,0 +1,112 @@
+/* START INTRODUCE */
+.introduce-content {
+  margin: 30px auto;  
+  width: 80%;
+  text-align: center;
+}
+
+.introduce-title {
+  line-height: 32.78px;
+  width: 100%;
+  margin-bottom: 30px;
+}
+
+.service-introduction {
+  width: 100%;
+  margin: 0;
+  line-height: var(--line-height-s);
+}
+
+.introduce-box {
+  width: 80%;
+  margin: 0 auto 80px;
+  text-align: center;
+}
+
+.introduce-icon {
+  margin-bottom: 10px;
+}
+
+.title-small {
+  margin-bottom: 10px;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.content-services {
+  width: 100%;
+  font-size: var(--font-size-m);
+  line-height: 28px;
+  letter-spacing: 0.2px;
+}
+
+/* TABLET */
+@media (min-width: 650px) {
+  .introduce-content {
+    margin-top: 60px;
+    margin-bottom: 60px;
+  }
+
+  .introduce-title {
+    line-height: 30px;
+  }
+
+  .introduce-box {
+    width: 80%;    
+    text-align: center;
+    margin: 99px auto 0;
+  }
+
+  .service {
+    margin: 0 auto 90px;
+  }
+
+  .content-services {
+    width: 60%;
+    margin: 0 auto;
+  }  
+}
+
+/* DESKTOP */
+@media (min-width: 1024px) {
+  .introduce-content {
+    width: 68%;
+    margin: 120px auto 108px;
+  }
+
+  .introduce-box {
+    width: 100%;
+    margin-bottom: 140px;
+  }
+
+  .introduce-title {
+    line-height: 66px;
+  }
+
+  .service-introduction {
+    line-height: var(--line-height-l);
+  }
+
+  .service {
+    width: 30%;
+    display: inline-block;
+    margin: 0 auto;
+  }
+
+  .mb-83 {
```

Comment:
This style should not be here. Btw, you should round this value to 85px, because 83px not use commonly